Makefile

```diff
@@ -117,6 +117,9 @@ integration.fedora:
 integration.centos:
 	DIST=CentOS sh .papr_prepare.sh
 
+shell: libpodimage
+	docker run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TESTFLAGS -e TRAVIS -it --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${LIBPOD_IMAGE} sh
```

Comment:
At some point we should consolidate these opts into a variable in the makefile

Reply:
It feels so wrong to run podman tests with docker, when podman could do it itself.

Reply:
at this point, it is just a commitment to do it.